templates/blacklight-compose.ecs.yml

```diff
@@ -18,6 +18,7 @@ services:
       RAILS_MASTER_KEY: ${RAILS_MASTER_KEY}
       SAMPLE_BUCKET:
       SOLR_URL: http://${CLUSTER_NAME}-solr.${CLUSTER_NAME}:8983/solr/blacklight-core
+      YALE_NETWORK_IPS: ${YALE_NETWORK_IPS:-10.0.0.0/8,172.16.0.0/12,192.168.0.0/16,127.0.0.1}
```

Comment:
since these are the same, this can be defined in blacklight-compose.yml instead of in both the ecs and local files. you only need these 2 files if you are overriding one or the other.

Reply:
Thanks. I updated to only add the setting to blacklight-compose.yml